man/systemd.special.xml

```diff
@@ -285,15 +286,17 @@
         <term><filename>hibernate.target</filename></term>
         <listitem>
           <para>A special target unit for hibernating the system. This
-          pulls in <filename>sleep.target</filename>.</para>
+          pulls in <filename>sleep.target</filename> and
+          <filename>wakeup.target</filename>.</para>
```

Comment:
Your patch doesn't actually do this though, as it hooks `wakeup.target` into the services, not the targets.